indra/llui/lltexteditor.h

```diff
@@ -313,12 +313,11 @@ class LLTextEditor :
     template <typename STRINGTYPE>
     void            pasteTextWithLinebreaks(const STRINGTYPE& clean_string)
     {
-        pasteTextWithLinebreaks<LLWString>(ll_convert(clean_string));
+        pasteTextWithLinebreaksInternal(ll_convert(clean_string));
     }
-    template <>
-    void            pasteTextWithLinebreaks<LLWString>(const LLWString & clean_string);
```

Comment:
This "explicit specialization in non-namespace context" does not build on linux.  My solution is to supply an explicit internal method for handling all input converted to `LLWString`.

Reply:
That may be the case but but I think the explicit impl is easier to read/understand.